src/templates/fr/2019/methodology.html

```diff
@@ -0,0 +1,724 @@
+{% extends "fr/2019/base.html" %}
+
+{% block title %}Méthodologie | Web Almanac par HTTP Archive{% endblock %}
+
+{% block description %}Décrit comment le Web
+Almanac 2019 a été conçu : Les ensembles de données et outils utilisés et la
+façon dont le projet a été exécuté.{% endblock %}
```

Comment:
Small thing, but can we keep the same line breaks? Makes it easier to keep any future changes in sync if on same line numbers.